APE0.rst

```diff
@@ -0,0 +1,393 @@
+The Astropy Project Governance Charter
+======================================
+
+author: The Astropy Governance Working Group; Tom Aldcroft, Matt Craig, Kelle Cruz, Lia Corrales, Steve Crawford, Nicole Foster, Pey Lian Lim, Stuart Mumford, Adrian Price-Whelan, Thomas Robitaille, David Shupe, Brigitta Sipocz, Erik Tollerud
+
+date-created: 2020 July 31
+
+date-last-revised: 2021 January 28
+
+date-accepted: <replace with accepted date>
+
+type: Process
+
+status: Discussion
+
+Abstract
+========
+This APE defines the formal fundamental governance process for the Astropy
+Project. The Astropy Project broadly follows the philosophy of do-ocracy:
+project participants who gain experience and knowledge through sustained
+community contributions and have demonstrated adherence to core principles of
+inclusion and consensus-building will be entrusted with increasing levels of
+authority. This peaks with the Coordination Committee, which has relatively
+broad authority but seeks to exercise it as rarely as possible by working
+towards consensus. This is because at its core, the Project is both by and for
+its community, and therefore the community is the final source of authority and
+direction.
+
+Detailed Description
+====================
+The Governance Charter is described in this section. The initial version is the
+outcome of several months of discussion by the Astropy Governance Working Group,
+which aimed to both codify existing informal governance rules as well as update
+them with new ideas that are consistent with the philosophy of and vision for
+the Astropy project.
+
+Ultimate Authority: The Astropy Community
+-----------------------------------------
+The Astropy Project derives its authority from the whole community - developers,
+users, and even potential participants in the broader astronomical and
+scientific software community. All have a vested interest in the success of the
+Project, so all governance in the end belongs to this community in its entirety.
+The governance Charter described below provides levels of delegation of that
+authority, particularly to the `Voting Members <votingmembers>`_ and the
+Coordination Committee.
+
+The Coordination Committee
+--------------------------
+
+Composition
+^^^^^^^^^^^
+The Coordination Committee is a committee that is composed of four or five
+members (this number will be decided by vote at the same time as the first
+Coordination Committee election). The
+Coordination Committee should aim to represent the diversity of the Astropy
+community.  Hence, it should aim for diversity in geography, skill set (both
+astronomy and developer), background, and so on. Furthermore, unless it would
```

Comment:
> skill set (both astronomy and developer)
This is too vague, either define a skill set enough to be on the coco or remove it. There was discussion above where it was preferred by many of us to remove all the vague language like this from this document. 

Similarly background, what type of background do you refer to, how that definition translate for a global project? How do you propose to require a disclosal of background info during an election? If data is not collected, and it's not disclosed how do you plan do aim for diversity along that angle?

Reply:
I agree, this is too vague. Here's my proposed re-write, which just cuts most of it:
`The Coordination Committee is a committee that is composed of four or five members (this number will be decided by vote at the same time as the first Coordination Committee election). Unless it would be short of candidates otherwise, it should not have more than one member from any particular employer.`

Reply:
I'm not thrilled about that change. the wording in the current text was an attempt to synthesize a wide range of opinions in this thread: https://github.com/astropy/astropy-APEs/pull/61#discussion_r502554623 .  I agree it's a bit vague, and impossible to set as a specific rule, but to me that was the point: these are things that are *impossible* to define fully, but making an attempt to account for them helps improve things. There was enough discussion on this both in the WG and the thread I linked above to make me thing it was important enough to write in there as an explicit desire, even if the exact definitions are not pinned down.

That said, if several of the people in that exchange (@mhvk, @hamogu, @adrn, @embray, @Cadair ) are fine with @kelle's proposed change, I'm ok with it too!  I just don't want to silence those voices at the last minute, since that's what led to this wording in the first place.

Reply:
I'm happy with the rephrase. 

And can also be happy if someone has a more concrete definition of skills and background, but I think it's super difficult and has to answer my question above to be effective and useful in practice.

Reply:
>but making an attempt to account for them helps improve things. 

Saying vague things with no consequences doesn't improve things. How do you propose to take background into account. What did you even mean by background here? If you don't collect data, how would you think it can be taken into account during an election?

Reply:
I think there is value in expressing intent, even if there are no consequences. It's quite common to express some intent even if the actual rules of the document don't spell out how that is achieved in detail (e.g. from a well-known text in the history of England "Know that we, out of reverence for God and for the salvation of our soul"). So I like it as long as the text makes clear that this is the *intent* not an enforceable rule. I think "it should aim" captures that quite well.
Since it's only an intent and not an enforceable rule it's OK not to collect the background, etc for statistical purposes. We say "we would like to have diversity and here are some examples" and not "we have a watertight definition of all axes of diversity".

However, if other people strongly disagree, I don't want to hold up APE0 over this. Getting the rules set is much more important.

Reply:
I don't think that the vagueness of a 800 year old document is relevant for an APE, which should be more about setting the rules, with actionable items.

Let's say there are X number of candidates. And a document that say we aim for diversity in background. What do you know, ideally have to know, about them to decide about your votes with a decision that aims to take diversity in background into consideration?  If you don't know any extra info about their background, or even a definition what you mean by background, then how can you even pretend to have an _intent_ for that diversity, beyond feeling good that the governance document is so great that it even covered this corner of unfairness and bias (albeit with nothing actionable).

It's the good old "women and underrepresented minorities are encouraged to apply", then business as usual, with all the shortlisted people are white male from US coast universities. The call had that phrase carrying the value, after all, they did everything they could (and then gender and race are probably easier to quantify for an application/nomination than "background" and "diverse skill set both astronomy and developer")

My original questions are still unanswered:
> what type of background do you refer to, how that definition translate for a global project? How do you propose to require a disclosal of background info during an election? If data is not collected, and it's not disclosed how do you plan do aim for diversity along that angle?

Reply:
>  what type of background do you refer to, how that definition translate for a global project? How do you propose to require a disclosal of background info during an election? If data is not collected, and it's not disclosed how do you plan do aim for diversity along that angle?

I would suggest to allow each candidate to write a CV / statement if they want. In that statement, they can disclose whatever they want but don't have to. Some things will be more obvious (geography) then others (socio-economic background). It's then up to the voters do do with that information whatever they want. In practice, most people who will stand to vote for Coco elections are probably reasonably well known to many in the community so we know from past actions about their skillset.

However, none of this *has* to be in the APE0 if it's contentions. So, let's leave that out.

Reply:
The whole long thread was started because there was a worry that CoCo would become a cabal, perhaps East Coast, dominated by STScI and other institutes that are able to pay their staff to work on astropy. But looking at it again it does seem a bit of a fool's errand to try to spell out the diversity we are aiming for, and even more to try to nail that down further.

I like @hamogu's suggestion of explicitly asking for a statement. How about:
```
The Coordination Committee should aim to represent the diversity of the Astropy community, and candidates are encouraged to include in their statement how they will contribute to the further diversification of ideas.
```
(last part directly stolen from my university's obligatory sentence -- I actually always liked that because it is spells out why diversity is good for the university).

But I'm also happy with just removing it altogether.

Reply:
Just another vote that I think it's fine to remove this - @bsipocz has a point that unless we actually do some kind of affirmative action or select the CoCo with entrofy (!) saying that we 'should' aim for this diversity doesn't actually mean anything as there is no enforcement. The only thing that would have an impact is if we actually included hard rules of e.g. no more than one CoCo member per continent, or some quotas on gender diversity (which is a minefield in itself given that gender is not a binary property).

@mhvk - I'd hope that the CoCo can be allowed to be more diverse in some ways than the astropy community - a sentence like what you suggests could actually be used to prevent too many underrepresented people on the CoCo if they are underrepresented in the astropy community!

Reply:
It ended up in a slightly different thread, but the consensus seemed to be pretty solidly behind the reword to what @kelle suggested, so I know when I'm out-voted :wink: